src/style.css

```diff
@@ -1,3 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
+  --blue-color: #00acdc;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  font-family: "Roboto", sans-serif;
+}
+
+.logo_link {
+  display: flex;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+}
+
+.list_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  padding: 23px 0;
+  text-transform: uppercase;
+}
```

Comment:
This is valid, but you are relying on padding + line-height to define the total height of the block, but line-height is not the font-size

In this situation, it is better to set line-height explicitly and not rely just on the font size

Alternatively, you can set the height for the `.nav_link` and center the text vertically